src/shader.cpp

```diff
@@ -871,3 +877,18 @@ void load_shaders(std::string name, SourceShaderCache *sourcecache,
 	}
 
 }
+
+void dumpShaderProgram(const std::string &program_type, const std::string &program)
+{
+	errorstream<<program_type<<" shader program:"<<std::endl<<
```

Comment:
Write the shader program to warningstream instead. errorstream gets printed into the chat in the client and must be very concise.